test/functional/tests/data_integrity/test_data_integrity_2d.py

```diff
@@ -40,11 +41,11 @@ def shuffled_fs_list(n):
 @pytest.mark.require_disk("core3", DiskTypeLowerThan("cache"))
 @pytest.mark.require_disk("core4", DiskTypeLowerThan("cache"))
 @pytest.mark.parametrize("filesystems", [shuffled_fs_list(4)])
-def test_data_integrity_5d_dss(filesystems):
+def test_data_integrity_2d_dss(filesystems):
     """
         title: |
           Data integrity test on three cas instances with different
-          file systems with duration time equal to 5 days
+          file systems with duration time equal to 2 days
         description: |
           Create 3 cache instances on caches equal to 50GB and cores equal to 150GB
```

Comment:
Remove mention of the number of cache instances - it's not important at this point and does not match the actual number of cache instances being started below.